src/forms/tests/manual/forms.html

```diff
@@ -397,5 +397,15 @@ <h2>Buttons</h2>
         <input type="button" class="pure-button pure-button-primary" value="Input Button" />
         <input type="reset" class="pure-button pure-button-primary" value="Reset" />
     </form>
+
+
+    <h2>File Uploads</h2>
+
+    <form class="pure-form">
+        <label class="pure-button" tabindex="0">
+            Browse <input type="file" hidden>
+        </label>
```

Comment:
I found [this HTML-only solution](https://stackoverflow.com/questions/11235206/twitter-bootstrap-form-file-element-upload-button), but it wasn't "tabbable." I added a `tabindex` to the label and now it is.

Reply:
So by making the input hidden, we lose the file name after selecting it?

Reply:
Correct.